docs/gettingStarted/plugins/gnuradio/Linux/buildLinux.rst

```diff
@@ -0,0 +1,62 @@
+Build from source
+=================
+
+Prerequisites
+-------------
+
+Required components to compile gnuradio-limesuiteng plugin:
+
+#. GNURadio (version 3.10.9.2)
+#. Boost    (version 1.83 or higher)
+#. Pybind11 (version 2.11.1 or higher)
+#. Gmp
+#. Python   (version 3.12.3 or higher)
+#. C++ compiler (GCC major version 13.xx)
+#. CMake (version 3.28.3 or higher)
+#. LimeSuiteNG
+
+.. Note::
+   Some plugin component versions may be incompatible and can break the functionality of a plugin. For example, higher versions of pybind11 can cause functionality problems as the GNURadio itself uses 2.11.1 version of pybind11. It is recommended to install the specified minimum version of component from required component list.
```

Comment:
Assuming the dependencies are installed using linux package manager, then there will be no versions incompatibility.